src/views/search_suggestions/search_suggestions_item_view.js

```diff
@@ -1,24 +1,9 @@
-import BaseView from '../base_view';
+import RowItemView from '../row_item_view';
 import SearchSuggestionsItemTemplate from '../../templates/search_suggestions/item.html';
-import webChannel from '../../lib/web_channel';
 
-export default BaseView.extend({
+export default RowItemView.extend({
   template: SearchSuggestionsItemTemplate,
 
-  events: {
-    'mousedown': 'openSuggestion',
-    'select': 'sendSelectionDetails'
-  },
+  events: RowItemView.prototype.events
```

Comment:
Sadly, `events` aren't inherited on views, so this is the awkward recommended approach. Have a look at the paragraph just above this point in the docs: https://github.com/AmpersandJS/ampersand-view#bindings